wear/src/main/res/layout/activity_contact_events.xml

```diff
@@ -18,26 +18,22 @@
       android:id="@+id/contact_events_date_text"
       style="@style/ContactEvents.Label.Date"
       android:layout_width="wrap_content"
-      android:layout_height="wrap_content"
-      android:drawablePadding="8dp"
-      android:drawableTop="@drawable/ic_complication_contact_event"
-      android:gravity="center" />
+      android:layout_height="wrap_content" />
 
     <TextView
       android:id="@+id/contact_events_names_text"
       style="@style/ContactEvents.Label.Names"
       android:layout_width="wrap_content"
       android:layout_height="wrap_content"
-      android:layout_marginTop="16dp"
-      android:gravity="center" />
+      android:layout_marginTop="16dp" />
   </LinearLayout>
 
   <TextView
     android:id="@+id/contact_events_empty_text"
+    style="@style/ContactEvents.Label"
     android:layout_width="wrap_content"
     android:layout_height="wrap_content"
     android:layout_centerInParent="true"
-    android:gravity="center"
```

Comment:
But I had: I actually forgot to apply the fontFamily before, so now using the style that's done as well, other than extracting the gravity